test/entry_test.py

```diff
@@ -1 +1,53 @@
-import entry
\ No newline at end of file
+import logging
+import os
+import shutil
+import unittest
+from autologging import logged
+from datetime import datetime
+from exam_date.stored_date import AssignmentLatestSubmittedDate
+
+logging.basicConfig(level=os.getenv("log_level", "DEBUG"))
+
+
+@logged
+class TestSPEProcess(unittest.TestCase):
+    @classmethod
+    def setUpClass(self):
+        self.__log.info("you got into the testing zone")
+        this_dir = os.path.dirname(os.path.abspath(__file__))
+        self.persisted_dir: str = this_dir + '/' + 'PERSIST/'
```

Comment:
Instead of using the current directory, write to "/tmp".  Then you won't need to tell git to ignore "test/PERSIST", either.

Reply:
Done